### PR TITLE
deps: Pin nmstate to 1.0-x

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:
The latest nmstate 1.2.0 has a bug that breaks k8s connectivity [1].
This change pin to a working version

[1] https://bugzilla.redhat.com/show_bug.cgi?id=2037411

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note
Pin nmstate to 1.1.0
```

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN env
 COPY build/_output/bin/handler.manager.linux-$TARGETARCH /usr/local/bin/manager
 
 RUN dnf install -b -y dnf-plugins-core && \
-    dnf install -b -y -x "*alpha*" -x "*beta*" nmstate && \
+    dnf install -b -y -x "*alpha*" -x "*beta*" nmstate-1.0.2 && \
```
Comment (phoracek) [inline build/Dockerfile:9]: The PR says 1.1 but here I see 1.0. Which one is wrong?

**Review by phoracek — CHANGES_REQUESTED** [review]

File: build/Dockerfile (hunk 1 above)
Comment (qinqon) [reply build/Dockerfile:9]: We have try 1.1.0 and it's failing so we pin at 1.0 but it's failing too.

**Comment by rhrazdil** [comment]
/hold

we shouldn't need this as soon as https://github.com/nmstate/kubernetes-nmstate/pull/958 is merged

**Pull request closed without merging**